desc/objectives/_neoclassical.py

```diff
@@ -54,12 +66,15 @@ class EffectiveRipple(_Objective):
     ----------
     eq : Equilibrium
         ``Equilibrium`` to be optimized.
-    rho : ndarray
-        Unique coordinate values specifying flux surfaces to compute on.
-    alpha : ndarray
-        Unique coordinate values specifying field line labels to compute on.
-    batch : bool
-        Whether to vectorize part of the computation. Default is true.
+    grid : Grid
+        Optional, tensor-product grid in (ρ, θ, ζ) with uniformly spaced nodes
+        (θ, ζ) ∈ [0, 2π) × [0, 2π/NFP). Powers of two are preferable.
+    X : int
+        Grid resolution in poloidal direction for Clebsch coordinate grid.
+        Preferably power of 2.
+    Y : int
```

Comment:
I don't want ``num_zeta`` because people would ask me how that's different than ``grid.num_zeta``, whether it uses the latter as a default, and would associate the resolution with the fourier resolution rather than chebyshev or spline